lib/taste_tester/host.rb

```diff
@@ -220,11 +228,11 @@ def config
       if TasteTester::Config.use_ssh_tunnels
         url = "#{scheme}://localhost:#{@tunnel.port}"
       else
-        url = "#{scheme}://#{@server.host}"
+        url = +"#{scheme}://#{@server.host}"
```

Comment:
this is ... bad syntax afact?

Reply:
Nope, mutable string: https://stackoverflow.com/a/37799399

Reply:
Oh jesus. This seems... onerous to me. but w/e